app/avo/resources/speaker.rb

```diff
@@ -16,6 +16,7 @@ def fields
     field :name, as: :text, link_to_record: true, sortable: true
     field :twitter, as: :text
     field :github, as: :text
+    field :speakerdeck, as: :text
```

Comment:
```suggestion
    field :speakerdeck, as: :text,  hide_on: :index
```

I think we should hide it from the index view

Reply:
I was thinking about that and then thought: "oh we have all the other socials here too"

![CleanShot 2024-10-20 at 23 24 56](https://github.com/user-attachments/assets/c6f2d270-ba8e-4f41-b3ec-e7b3fdb98c65)

Reply:
yeah it's just that now the actions (show/edit ...) are out of the viewport

will try it and update it if needed